src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreJob.cs

```diff
@@ -683,7 +695,8 @@ private bool AnyProjectHasVulnerablePackageWarning(IReadOnlyList<RestoreSummary>
                             // Display the restore opt out message if it has not been shown yet
                             await l.WriteHeaderAsync();
 
-                            await RestoreMissingPackagesInSolutionAsync(solutionDirectory, packages, l, t);
+                            PackageRestoreResult packageRestoreResult = await RestoreMissingPackagesInSolutionAsync(solutionDirectory, packages, l, t);
```

Comment:
I would love to see better variable names than `l` and `t` but that can be a follow-up...